doc/ref_model/chapters/appendix-a.md

```diff
@@ -1,5 +1,5 @@
 [<< Back](../../ref_model)
-# Appendix A - Guidelines for VNF Vendors
+# Appendix A - Guidelines For Application Vendors
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
```

Comment:
@kedmison we are using Workload not application, what did you think

Reply:
We have a lot of mixed use of VNF, Workload, and Application.  I think we generally perceive workload and application as consistent with each other, but VNF is perhaps a bit too specific relative to VM-based virtualization.  
So, I think it's acceptable.